src/Brie.h

```diff
@@ -2043,7 +2043,7 @@ template <unsigned Pos, unsigned Dim>
 struct fix_binding<0, Pos, Dim> {
     template <unsigned bits, typename iterator, typename entry_type>
     bool operator()(
-            const SparseBitMap<bits>& store, iterator& begin, iterator& end, const entry_type& entry) const {
+            const SparseBitMap<bits>& store, iterator& begin, iterator& /* end */, const entry_type&) const {
```

Comment:
Our consistency overall needs work, but try to be consistent within the same line
```suggestion
            const SparseBitMap<bits>& store, iterator& begin, iterator& /* end */, const entry_type& /* entry */) const {
```

Reply:
I usually omit the variable name whenever its purpose is unambiguous from its type. e.g. `iterator` requires a commented name as it is unclear whether it is the start or end of a range, while `entry_type` does not since its purpose can be reliably inferred from its type.